SU-HKKU/cardinal_service_profile
# CSD-362 Updated opportunity row and page

# READY FOR REVIEW

# Summary
- Updated opportunity row and page

# Needed By (Date)
- 08/31

# Urgency
- mediuma

# Steps to Test

1. Pull this PR
2. Clear cache
3. Check for the following updates:
- Opportunity thumbnail should now have card border and shadow
- Opportunity pages should now include CS Program, Type of Opportunity, H1, then Commitment and Service theme tags.
- Opportunity card (containing when, location, etc) should have `modular-spacing(4)`
- Spotlight view image should be `width: 250px`
- Only Related Resource and Opportunity list titles should be `text-align: center`
- Opportunity list paragraph should not have any list styles


# Associated Issues and/or People
- [CSD-362](https://stanfordits.atlassian.net/browse/CSD-362)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
